cardano-cli/src/Cardano/CLI/EraBased/Run/Transaction.hs

```diff
@@ -532,7 +530,6 @@ runTxBuild
     <- hoistEither $ first TxCmdReturnCollateralValidationError $ validateTxReturnCollateral era mReturnCollateral
   dFee <- hoistEither $ first TxCmdTxFeeValidationError $ validateTxFee era dummyFee
   validatedLowerBound <- hoistEither (first TxCmdTxValidityLowerBoundValidationError (validateTxValidityLowerBound era mLowerBound))
-  validatedUpperBound <- hoistEither (first TxCmdTxValidityUpperBoundValidationError (validateTxValidityUpperBound era mUpperBound))
```

Comment:
We no longer do validation here because by virtue of using era-sensitive types in the era-based CLI parser the values are already valid.

Reply:
Does this mean I should remove this line `validateUpperBound <-... ` ?